x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/translations.ts

```diff
@@ -196,3 +197,25 @@ export const GENERATION_ERROR = (progressStep: string) =>
 export const RETRY = i18n.translate('xpack.integrationAssistant.step.dataStream.retryButtonLabel', {
   defaultMessage: 'Retry',
 });
+
+export const ERROR_TRANSLATION = {
```

Comment:
To make sure every `ErrorCode` is mapped to a translation we need to type it:
```suggestion
export const ERROR_TRANSLATION: Record<ErrorCode, string> = {
```

Reply:
Ohh yes.. Thanks